### ISSUE TITLE
[Bug]:  nextCmd from pysnmp.hlapi.asyncio not return a generator

### ISSUE THREAD

**Issue by Etibru** [lextudio/pysnmp#55]
### Expected behavior

In the documentation on nextCmd (from pysnmp.hlapi.asyncio) it is written that nextCmd Creates a generator to perform SNMP GETNEXT query

### Actual behavior

But a bug happened like this, and the nextCmd the return type list

### Detailed steps

```py
error_indication, _, _, var_bind_table = await nextCmd(
            self.snmp_engine,
            self.user,
            UdpTransportTarget((self.ip, 161), timeout=1, retries=1),
            ContextData(),
            ObjectType(ObjectIdentity(oid)),
            lexicographicMode=False,
        )
        print("type ", type(var_bind_table))
```
==> `type <class 'list'>`

For more details, I'm using pysnmp-lextudio in async and I'd like to retrieve as a snmpwalk all the children of an oid. 
example: 
1.2.3.4.5.1

return : 1.2.3.4.5.1.1, 1.2.3.4.5.1.2, 1.2.3.4.5.1.3, .... but stop before 1.2.3.4.5.2

### Python package information

6.0.6

### Operating system information

Linux

### Python information

3.11

**Comment by lextm** [comment]
We currently consider this a documentation mistake but we will continue investigating.

**Comment by Etibru** [comment]
Okay, thanks.
Maybe I'm going about this the wrong way and there's another way to snmpwalk in asyncio?

**Comment by lextm** [comment]
After reviewing 4.4.x releases, we confirmed that asyncio API was originally designed this way.

We will fix the documentation error soon, and consider some new API to help perform WALK operations.

**Comment by lextm** [comment]
Just shipped `walkCmd` and `bulkWalkCmd` in release 6.0.7. You can use them to replace the asyncore based `nextCmd` and `bulkCmd`.
